android/src/main/kotlin/com/situm/situm_flutter_wayfinding/Extensions.kt

```diff
@@ -1,52 +1,39 @@
 package com.situm.situm_flutter_wayfinding
 
 import es.situm.sdk.error.Error
-import es.situm.sdk.model.cartography.Geofence
+import es.situm.sdk.model.MapperInterface
 import es.situm.sdk.model.cartography.Poi
 import es.situm.sdk.model.cartography.PoiCategory
-import es.situm.sdk.model.cartography.Point
 import io.flutter.plugin.common.MethodChannel
 
-fun List<Geofence>.toGeofencesMap(): List<Map<String, String>> {
+fun List<MapperInterface>.toMap(): List<Map<String, Any>> {
     return map {
-        mapOf( // Create a map for each Geofence.
-            "id" to it.identifier,
-            "name" to it.name
-        )
+        it.toMap()
     }
 }
 
 fun Collection<Poi>.toPoisMap(): List<Map<String, Any>> {
     return map {
-        mapOf( // Create a map for each Poi.
-            "id" to it.identifier,
-            "name" to it.name,
-            "buildingId" to it.buildingIdentifier,
-            "poiCategory" to it.category.toMap(),
-            "position" to it.position.toMap(),
-        )
+        mapOf(
+            // Create a map for each Poi.
+            // TODO: pending category identifier, then delete this method.
+            "poiCategory" to it.category.ftToMap(),
```

Comment:
Porqué se queren ignorar os campos de Poi e únicamente mappear o poiCategory ?

Reply:
Non se ignoran, engádense abaixo usando `it.toMap()`

Reply:
Non se ignoran, engádense abaixo usando `it.toMap()`